stats/view/view_to_metric_test.go

```diff
@@ -516,6 +517,85 @@ func TestUnitConversionForAggCount(t *testing.T) {
 	}
 }
 
+type mockExp struct {
+	metrics []*metricdata.Metric
+}
+
+func (me *mockExp) ExportMetrics(ctx context.Context, metrics []*metricdata.Metric) error {
+	me.metrics = append(me.metrics, metrics...)
+	return nil
+}
+
+var _ metricexport.Exporter = (*mockExp)(nil)
+
+func TestViewToMetric_OutOfOrderWithZeroBuckets(t *testing.T) {
+	m := stats.Int64("OutOfOrderWithZeroBuckets", "", "")
+	v := &View{
+		Measure:     m,
+		Aggregation: Distribution(10, 0, 2),
```

Comment:
Can you add a test with bucket bounds like [0, 5, 10]?

Reply:
done.